integration/e2ehistograms/e2ehistograms.go

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+//go:build requires_docker
```

Comment:
Does this package still need this `go:build` directive?

Reply:
Good point, removed.